Scripts/gen_CosmicStage2.sh

```diff
@@ -31,6 +32,7 @@ usage() {
   [ --low Resample 'Low' S1 output ]
   [ --owner (opt) default mu2e ]
   [ --dsstops (opt) expllicit list of DS stop files ]
+  [ --setup (opt) expllicit simjob setup ]
```

Comment:
I like this option, it makes testing easier.  We should follow this pattern as we update other scripts for mu2ejobsub.